uptime-kuma.json

```diff
@@ -56,72 +219,91 @@
             {
               "options": {
                 "0": {
-                  "color": "red",
+                  "color": "dark-red",
                   "index": 0,
-                  "text": "DOWN"
+                  "text": "Down"
                 },
                 "1": {
-                  "color": "green",
+                  "color": "dark-green",
+                  "index": 2,
+                  "text": "Up"
+                },
+                "2": {
+                  "color": "dark-yellow",
                   "index": 1,
```

Comment:
```suggestion
                  "index": 2,
```

Reply:
Index likely means that this is the ordering.
Let's keep the ordering red-yellow-green and just change the colours and indexes instead ^^